android/src/main/java/com/reactnativecommunity/netinfo/ConnectivityReceiver.java

```diff
@@ -208,7 +208,7 @@ private WritableMap createDetailsMap(@Nonnull String detailsInterface) {
                 }
                 break;
             case "wifi":
-                if (NetInfoUtils.isAccessWifiStatePermissionGranted(getReactContext())) {
+                if (NetInfoUtils.isAccessWifiStatePermissionGranted(getReactContext()) && mWifiManager != null) {
```

Comment:
This one is okay as you propose it, because details does not necessarily contain any extra information - so just skipping and not adding anything to details does not break expectations 👍